pkg/render/manager.go

```diff
@@ -264,9 +264,6 @@ func (c *managerComponent) managerDeployment() *appsv1.Deployment {
 
 // managerVolumes returns the volumes for the Tigera Secure manager component.
 func (c *managerComponent) managerVolumeMounts() []corev1.VolumeMount {
-	if c.cfg.KeyValidatorConfig != nil {
```

Comment:
I'm sure it's removed because it's unneeded, but could you explain why we can remove this please? It's not immediately intuitive to me.

Reply:
The required volume that was previously mounted contained the certificate that Dex presents. It is no longer required, because in `pkg/controller/manager/manager_controller.go`, this certificate is added to the trusted bundle.

Reply:
I did some digging. And thanks to you I found out that we cannot remove this. It is needed for OIDCTypeTigera, which was implemented for an edge case with a customer (and maybe also used in cloud).
However, we can modify this. We do not need the Volume, because it is already there, but we do need the mount somewhere in the /etc/ssl/certs folder, which is alpines location for trusted certificates.
Thank you!